docs/source/noise_dataset.ipynb

```diff
@@ -42,47 +42,73 @@
+    "Similarly to the `WaveformDataset`, the `ASDDataset` is just a container. Dingo includes routines for building such a dataset from observational data.\n",
     "\n",
-    "1. Identify stretches of data within an observing run meeting certain criteria (sufficiently long, without events, and sufficiently high quality, ...).\n",
-    "2. Download data from [GWOSC](https://www.gw-openscience.org). \n",
+    "## Command-line scripts\n",
+    "\n",
+    "### `dingo_generate_asd_dataset`\n",
+    " The basic approach is as follows:\n",
+    "1. Identify stretches of data within an observing run meeting certain criteria (sufficiently long, without events, and sufficiently high quality, ...) or take-in user-specified stretches.\n",
+    "2. Fetch data corresponding to these stretches using either\n",
+    "    - [GWOSC](https://www.gw-openscience.org)\n",
+    "    - channels, optionally specified in the settings file.\n",
     "3. Estimate ASDs using Welch's method on these stretches.\n",
     "4. Save the collection of ASDs.\n",
     "\n",
-    "To generate an ASD dataset, execute the following script:\n",
-    "```bash\n",
-    "dingo_generate_ASD_dataset --data_dir data_dir --settings_file asd_dataset_settings.yaml\n",
-    "```\n",
+    "```text\n",
+    "usage: dingo_generate_asd_dataset [-h] --data_dir DATA_DIR [--settings_file SETTINGS_FILE] [--time_segments_file TIME_SEGMENTS_FILE] [--out_name OUT_NAME] [--verbose]\n",
+    "\n",
+    "Generate an ASD dataset based on a settings file.\n",
     "\n",
+    "optional arguments:\n",
+    "  -h, --help            show this help message and exit\n",
+    "  --data_dir DATA_DIR   Path where the PSD data is to be stored. Must contain a 'settings.yaml' file.\n",
+    "  --settings_file SETTINGS_FILE\n",
+    "                        Path to a settings file in case two different datasets are generated in the same directory\n",
+    "  --time_segments_file TIME_SEGMENTS_FILE\n",
+    "                        Optional file containing a dictionary of a list of time segments that should be used for estimating PSDs.This has to be a pickle file.\n",
+    "  --out_name OUT_NAME   Path to resulting ASD dataset\n",
+    "  --verbose\n",
+    "\n",
+    "```\n",
     "where the settings file is of the form\n",
     "```yaml\n",
     "dataset_settings:\n",
     "  f_s: 4096\n",
-    "  T_PSD: 1024\n",
+    "  time_psd: 1024\n",
     "  T: 8\n",
+    "  time_gap: 0\n",
     "  window:\n",
     "    roll_off: 0.4\n",
     "    type: tukey\n",
-    "  T_gap: 0\n",
-    "  num_psds_max: 2\n",
+    "  num_psds_max: 20\n",
+    "  channels:\n",
+    "   H1: H1:DCS-CALIB_STRAIN_C02\n",
+    "   L1: L1:DCS-CALIB_STRAIN_C02\n",
     "  detectors:\n",
     "    - H1\n",
     "    - L1\n",
     "  observing_run: O2\n",
+    "condor:\n",
+    "  env_path: path/to/environment\n",
+    "  num_jobs: 2    # per detector\n",
+    "  num_cpus: 16\n",
+    "  memory_cpus: 16000\n",
+    "  bid: 200\n",
```

Comment:
Does the `bid` ever get used? I couldn't find it in the code. Also this is specific to the MPI-IS cluster, so it should be optional.

Reply:
True, I don't think it does anymore